MetaMask/metamask-mobile
# fix: migration to enable Blockaid by default

## **Description**



Migrating from a previous version to v7.17.0 should enable Blockaid by default in the same way it's done with 7.17.0 fresh installs.

## **Related issues**

Fixes: #8669 

## **Manual testing steps**

1. Install v7.16
2. Update to v7.17
3. Check settings --> see Blockaid is enabled


## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
